acceptance/suites/pre_suite/foss/80_configure_puppet.rb

```diff
@@ -6,4 +6,12 @@
   lay_down_new_puppet_conf( master,
                            {"main" => { "dns_alt_names" => "puppet,#{hostname},#{fqdn}",
                                        "verbose" => true }}, dir)
+
+  on master, %q{/opt/puppetlabs/puppet/bin/ruby -e } +
+             %q{'require "hocon/parser/config_document_factory"; } +
+             %q{file = "/etc/puppetlabs/puppetserver/conf.d/puppetserver.conf"; } +
+             %q{o = Hocon::Parser::ConfigDocumentFactory.parse_file(file); } +
+             %q{n = o.set_value("certificate-authority.allow-subject-alt-names", "true"); } +
+             %q{File.write(file, n.render)'}
```

Comment:
We should see if we can use [modify_tk_config](https://github.com/puppetlabs/beaker-puppet/blob/2f05918b6689da1210f656557ef671f591b386d3/lib/beaker-puppet/helpers/tk_helpers.rb#L28) here.

Reply:
Looks like we should have it available, testing an update to use it now.